### ISSUE TITLE
Is there any way to use with 'karma-coverage-istanbul-reporter' ?

### ISSUE THREAD

**Issue by black-trooper** [caitp/karma-coveralls#44]
I want to use with 'karma-coverage-istanbul-reporter'.
```js
reporters: ['coverage-istanbul', 'coveralls']
```

However, an error occurred.
```
"Coverall" can not be loaded!
Error: coverage reporter should precede coveralls
```

**Comment by caitp** [comment]
if you send a pull request, it can be easily supported!

**Comment by caitp** [comment]
Your fix is published in v2.1.0
